docs/features/wait/sql.md

```diff
@@ -0,0 +1,22 @@
+# SQL Wait strategy
+
+The SQL wait strategy will check the result of an HTTP(S) request that is available in the container, being able to set the following conditions:
```

Comment:
Of an HTTP request? I assume of a SQL query?

Reply:
Wrong copy&paste? 🤦  Changing it...

Reply:
Done